src/gadgets/utils.rs

```diff
@@ -70,6 +70,7 @@ pub fn alloc_one<F: PrimeField, CS: ConstraintSystem<F>>(mut cs: CS) -> Allocate
 
 /// alloc a field as a constant
 /// implemented refer from <https://github.com/lurk-lab/lurk-rs/blob/4335fbb3290ed1a1176e29428f7daacb47f8033d/src/circuit/gadgets/data.rs#L387-L402>
+#[allow(unused)]
```

Comment:
We should bin it I think, this is a Lurk addition not in Nova main. And we have copies of it lying around.

Reply:
Removed it, but the function was used in tests. The commit replaces their usages with other existing functions